x-pack/plugins/rule_registry/server/utils/create_lifecycle_rule_type.test.ts

```diff
@@ -109,7 +118,7 @@ function createRule() {
         tags: ['tags'],
         updatedBy: 'updatedBy',
         namespace: 'namespace',
-      });
+      })) ?? {}) as Record<string, any>;
```

Comment:
@ecezalp I left this as they had it for now. It would be easy for them to adopt a pattern like ours (which allows static typing of `state`, but I didn't want to make that decision for them.